src/scripts/main.js

```diff
@@ -1,5 +1,18 @@
 'use strict';
 
+let wall = document.querySelector('.wall');
+let spider = document.querySelector('.spider');
+let wallOffsetLeft = wall.offsetLeft;
+let wallOffsetTop = wall.offsetTop;
+
 document.addEventListener('click', (e) => {
-  // write code here
+  if (e.target.className !== 'wall') {
+    return;
+  }
+
+  if (e.target.className === 'wall') {
```

Comment:
The condition `if (e.target.className !== 'wall')` is followed by another check `if (e.target.className === 'wall')`. This is redundant because if the first condition is false, the second condition will always be true. You can remove the second condition and directly execute the code inside it.